app/gui/src/ide.rs

```diff
@@ -28,6 +28,8 @@ pub use initializer::Initializer;
 /// Text that shows up in the statusbar when any of the backend connections is lost.
 pub const BACKEND_DISCONNECTED_MESSAGE: &str =
     "Connection to the backend has been lost. Please try restarting IDE.";
+/// Text that shows up in the statusbar when backend reports a failed execution.
+pub const EXECUTION_FAILED_MESSAGE: &str = "Execution failed. Please try restarting IDE.";
```

Comment:
We could also display the message received from the engine, making debugging user problems easier.

The question is if the messages are not too long.

Reply:
"Please report this problem at `support@enso.org`" as well?

Reply:
The message from the engine seems to be too long. I added the email address as suggested